app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer_test.go

```diff
@@ -601,18 +605,35 @@ func TestRemoveClustersFromStatus(t *testing.T) {
 	}
 	for i, c := range testCases {
 		glog.Infof("===============testing case: %d=================", i)
+		typedFRS := frs.(*ForwardingRuleSyncer)
+
 		if c.http {
 			// Ensure http forwarding rule.
-			if err := frs.EnsureHttpForwardingRule(lbName, ipAddr, tpLink, clusters, false /*force*/); err != nil {
+			if err := frs.EnsureHttpForwardingRule(lbName, ipAddr, tpLink, false /*force*/); err != nil {
 				t.Errorf("expected no error in ensuring http forwarding rule, actual: %v", err)
 			}
+			name := typedFRS.namer.HttpForwardingRuleName()
+			// Add status to the forwarding rule to simulate old firewall rule which has status.
```

Comment:
s/old firewall/old forwarding/ ?

Reply:
Thanks. Found the same mistake at a few more places, including the title of this PR :)